Source/Core/Core/PrimeHack/Mods/FpsControls.h

```diff
@@ -28,7 +38,16 @@ class FpsControls : public PrimeMod {
     std::array<std::tuple<int, int>, 4> const &visors);
   void mp3_handle_cursor(bool lock);
 
+
   void run_mod_menu(Region region);
+  // Added by LTSchmiddy:
+
+  bool beam_visor_menu_handler(u32 cursor_base, u32 yaw_vel_address, Game game);
+  bool mp3_visor_menu_handler(u32 cursor_base, u32 yaw_vel_address);
+  //FpsControls::Beam_Visor_Menu_State beam_visor_menu_next_state(Beam_Visor_Menu_State state);
```

Comment:
Please go through your code to ensure it is clean, for instance unnecessary comments and too many empty lines.